tests/tests/tier0/bluechi-agent-config-c-option/test_agent_config_c_option.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+import logging
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+from bluechi_test.util import read_file
+
+LOGGER = logging.getLogger(__name__)
+
+NODE_FOO = "node-foo"
+active_status = "active"
+failed_status = "failed"
+
+
+def restart_with_config_file(config_file_location, container, service):
+    container.exec_run(f"sed -i '/ExecStart=/c\\ExecStart=/usr/libexec/{service} -c "
+                       f"{config_file_location}' "
+                       f"/usr/lib/systemd/system/{service}.service")
+    container.exec_run("systemctl daemon-reload")
+    container.exec_run(f"systemctl restart {service}.service")
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+
+    node_foo = nodes[NODE_FOO]
+    config_files_agent = "config-files-agent"
```

Comment:
To have the same file structure for all `-c` related tests I'd renamed the directory to `config-files` and have there relavant file names (for example `agent-invalid.conf`)